app/views/transcription_field/_field_layout.html.slim

```diff
@@ -14,32 +14,17 @@
           -span_width = !field.percentage.blank? ? field.percentage : @width
           .field-wrapper(style="width: #{span_width}%")
             -if field.input_type == "instruction"
-              div id="fields[#{field.id}][#{field.label}]" class="field-instructions"
+              div id=formatted_field_name(field) class="field-instructions"
                 h5 =t('.instructions')
-                p ==field.label
+                p ==field.label.titleize
             -else
-              =label_tag field.label, field.label
-              -content = cell.nil? ? nil : cell.content
-              -if field.input_type == "text"
-                =text_field_tag "fields[#{field.id}][#{field.label}]", content, class: 'field-input'
-              -elsif field.input_type == "date"
-                =text_field_tag "fields[#{field.id}][#{field.label}]", content, class: 'field-input edtf', data: { inputmask: '"alias": "datetime", "inputFormat": "isoDate"'}
-              -elsif field.input_type == "select"
-                -options = field.options.split(";") unless field.options.nil?
-                =select_tag("fields[#{field.id}][#{field.label}]", options_for_select(options, content), class: 'field-input')
-              -elsif field.input_type == "description"
-                =hidden_field_tag "fields[#{field.id}][#{field.label}]", content, class: 'field-input'
-              -elsif field.input_type == "alt text"
-                =text_area_tag "fields[#{field.id}][#{field.label}]", content, class: 'field-input alt-text', maxlength: '255'
-                div.character-count
-              -else
-                =text_area_tag "fields[#{field.id}][#{field.label}]", content, class: 'field-input'
+              =generate_field_input(field, cell)
```

Comment:
I moved some of the logic here to the helper function.

Reply:
Great!